Client/src/Components/product-reviews/styles.css

```diff
@@ -40,3 +55,73 @@
   margin: 20px;
   text-align: left;
 }
+
+.reviewsAndRating {
+  display: flex;
+}
+
+.starsOuter {
+  display: inline-block;
+  position: relative;
+  font-family: FontAwesome;
+}
+
+.starsOuter::before {
+  content: "\f006 \f006 \f006 \f006 \f006";
+}
+
+.starsInner {
+  position: absolute;
+  top: 0;
+  left: 0;
+  white-space: nowrap;
+  overflow: hidden;
+  width: 50%;
+}
+
+.starsInner::before {
+  content: "\f005 \f005 \f005 \f005 \f005";
+  color:black;
+}
+
+.barContainer {
+  width:50%;
+  background-color: #f1f1f1;
+  color: white;
+  display: inline-block;
+}
+
+.bar5 {
```

Comment:
Why are there multiple bar classes? They seem to be identical.

Reply:
Each star bar on the rating a review is independent of others. So for example, there could be four 5 star reviews and one 4 star review. In that case the css for the 5 star bar would be different from the 4 star bar so we need a separate bar CSS for each rating.